### ISSUE TITLE
Question about fablo's functionalities

### ISSUE THREAD

**Issue by nkaramolegos** [hyperledger-labs/fablo#282]
Hello, I would like to ask:
1) Can I use fablo for deploying a network in Kubernetes?
2) Can I use fablo to manage the network. By managing I mean enroll/register users, create channels, deploying chaincodes, query the ledger using the REST API?
3) How can I deploy a gateway client application (or [this](https://github.com/hyperledger/fabric-samples/tree/main/asset-transfer-basic/rest-api-typescript) which use REST) to query the ledger?

**Comment by Hejwo** [comment]
Hello, thanks for using Fablo! :)

> 1. Can I use fablo for deploying a network in Kubernetes?  

Not yet, we're slowly getting there. You can monitor [this issue](https://github.com/hyperledger-labs/fablo/issues/219) to be updated. 

> 2. Can I use fablo to manage the network. By managing I mean enroll/register users, create channels, deploying chaincodes, query the ledger using the REST API?

Partially yes. For now you can:  
- Use [fablo rest](https://github.com/softwaremill/fablo-rest) to enroll/register users, invoke/query chaincodes. Just add `"tools": { "fabloRest": true }` to organisation you want to expose and integrated fablo-rest will be started. 
- to change the network you must edit your `fablo-config.json` and then type `fablo recreate` to change number of peers, orderers, channels, etc.  
(But bare in mind that it will recreate your network and doesn't preserve previous state.)  

> 3. How can I deploy a gateway client application (or this which use REST) to query the ledger?  

You can do it with already mentioned [fablo rest](https://github.com/softwaremill/fablo-rest). On project's repo you have some guidelines and [here](https://github.com/softwaremill/fablo-rest/blob/main/e2e/happyPath.test.ts) you've got some sample code.
  
  
  
If something more is unclear don't hesitate to ask:)

**Comment by nkaramolegos** [comment]
Thank you for your answer. It seems that Fablo supports a lot of operations. The only problem is that in order to add a new channel (where the existing orgs will join) you have to recreate the entire network without keeping the state. This is a problem for my use case scenario.

**Comment by Hejwo** [comment]
yeah, unfortunately we're not supporting this kind of operation in fablo-rest yet.
Another tip might be: Starting your initial network with fablo and then spinning by yourself [mircofab](https://github.com/IBM-Blockchain/microfab) which provides advanced restful api, there's a good chance that it also supports adding new channel.

**Comment by dzikowski** [comment]
Closing: Cleanup of stale issues. Please reopen if needed.
